test_scripts/API/Navigation/commonNavigation.lua

```diff
@@ -14,7 +14,7 @@ local commonSteps = require("user_modules/shared_testcases/commonSteps")
 local commonTestCases = require("user_modules/shared_testcases/commonTestCases")
 local SDL = require('SDL')
 
---[[ Local Variables ]]
```

Comment:
@dboltovskyi Why did you remove `local`?

Reply:
@AByzhynar Because some of the variables are not local